lib/index.js

```diff
@@ -6,44 +6,30 @@ const Server = require('./server')
 const Socket = require('./socket')
 const EthApi = require('./eth-api')
 const Database = require('./database')
-
+const logger = require('../logger')
 const TokenExporter = require('./metronome/token-exporter')
+const ACCExporter = require('./metronome/acc-exporter')
 const Status = require('./metronome/status')
 const StatusCollector = require('./metronome/status-collector')
-
+const config = require('config')
 class MetApi {
-  constructor (config, logger) {
+  constructor () {
     this.logger = logger
-
     this.isRunning = false
     this.config = config
     this.database = new Database(config, this.logger)
     this.server = new Server(config, this.logger, this.database, this)
-
     this.socket = new Socket(config, this.logger, this.server.httpServer)
-
     if (config.eth.enabled) {
-      this.ethApi = new EthApi(config.eth, this.logger, () => this._initMetronome())
+      this.ethApi = new EthApi(this.config.eth, this.logger, () => this._initMetronome())
     }
   }
 
   _initMetronome () {
-    this.tokenExporter = new TokenExporter({
-      db: this.database,
-      config: this.config.eth,
-      logger: this.logger,
-      ethApi: this.ethApi,
-      socket: this.socket
-    })
-
-    this.status = new Status({
-      logger: this.logger,
-      db: this.database,
-      ethApi: this.ethApi,
-      socket: this.socket
-    })
-
-    this.statusCollector = StatusCollector.start()
+    this.tokenExporter = new TokenExporter(this.database, this.socket)
+    this.accExporter = new ACCExporter(this.database, this.socket)
+    // this.status = new Status(this.database, this.socket)
+    // this.statusCollector = StatusCollector.start()
```

Comment:
@rokso  we will need to uncomment and work as is.

Reply:
There are 2 commits can you more sure, you are checking latest changes.

Reply:
Resolved, as I end up squashing multi author commits.